lib/mpl_toolkits/basemap/__init__.py

```diff
@@ -3206,6 +3206,17 @@ def set_axes_limits(self,ax=None):
             import matplotlib.pyplot as plt
             plt.draw_if_interactive()
 
+
+    def _save_use_hold(self, ax, kwargs):
+        h = kwargs.pop('hold', None)
+        if hasattr(ax, '_hold'):
+            self._tmp_hold = ax._hold
+            ax._hold = h
```

Comment:
This assignment only happens if the axes already has a `_hold` attribute. So, this wouldn't work if someone was setting the hold initially.